daphne_worker/src/lib.rs

```diff
@@ -512,6 +519,70 @@ pub(crate) struct InternalTestAddTask {
     task_expiration: Time,
 }
 
+/// LogWriter helps us write to the worker console.
+///
+/// It provides and io::Write implementation that provides line
+/// buffering.  It also takes care of emitting a timestamp as the
+/// timestamp code in the tracing library tries to use standard
+/// clock code, which does not work in a worker.
+struct LogWriter {
+    buffer: String,
+}
+
+impl LogWriter {
+    fn new() -> Self {
+        LogWriter {
+            buffer: String::new(),
+        }
+    }
+}
+
+impl io::Write for LogWriter {
+    fn write(&mut self, buf: &[u8]) -> io::Result<usize> {
+        let s = str::from_utf8(buf).map_err(|e| io::Error::new(io::ErrorKind::InvalidInput, e))?;
+        self.buffer.push_str(s);
+        // We assume the caller is doing line buffering and that any write containing a NL will have it at the end.
+        if self.buffer.ends_with('\n') {
+            let mut date = Date::now().to_string();
+            // The format we get ends with the long time zone name in parens, e.g. " (Pacific Standard Time)".
+            // This is noisy, so if we see it, chop it off.
+            if let Some(index) = date.find(" (") {
+                date.truncate(index);
+            }
```

Comment:
That's an aspect of the javascript environment I think.  I'm not sure I get to pick or can tell, but maybe!  I was hoping that a metal would just be set to UTC and it would use that.  But in the meantime the format it uses was making me have an excessively wide terminal and not adding value, so I hacked it away

Reply:
Got it! Yeah it's fine to leave as-is for now. Definitely keep the comment for reference later.